lms/security.py

```diff
@@ -173,6 +193,20 @@ def _get_user(request):
 
 
 def includeme(config):
-    config.set_security_policy(SecurityPolicy())
+    config.set_security_policy(
+        SecurityPolicy(
+            PathPolicyGetter(
+                path_policy_mapping={
+                    # Paths are tested in sequence so the order here
+                    # is relevant.
+                    # More specific routes should be higher than shorter ones.
+                    "/admin": LMSGoogleSecurityPolicy,
+                    "/googleauth": LMSGoogleSecurityPolicy,
+                    # Fallback for the rest
+                    "/": LTISecurityPolicy,
```

Comment:
I think you probably want the values here to be instances not classes. A security policy class may want to use `self` to store state and may get caught out when it doesn't realise that there are multiple instances of itself. It would also mean that the `__init__()`'s of security policies could have different arguments and that could be handled here in the `includeme()`, rather than them all having to have the same interface.